.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/comit
+    machine:
+      image: ubuntu-1604:201903-01
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "Cargo.lock" }}
+      - setup_system
+      - setup_nodejs
+      - setup_rust
+      - print_current_versions
+      - run:
+          name: Build using cargo make
+          command: |
+            source $HOME/.cargo/env
+            cargo --version
+            cargo make ci
+      - save_cache:
+          key: dependency-cache-{{ checksum "Cargo.lock" }}
+          paths:
+            - "~/.cargo"
+            - "./target"
+
+
+commands:
+  setup_system:
+    steps:
+       - run:
+           name:  "Setup system environment"
+           command: |
+             wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+             curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+             echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+             sudo apt-get update
+             sudo apt-get install -y libzmq3-dev
+  setup_nodejs:
+    steps:
+       - run:
+           name:  "Install nodejs"
+           command: |
+             set +e
+             curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash
```

Comment:
:x: Why can't we just the same thing as in `travis` and install the correct node version from `apt`?

Reply:
I was unfortunately not able to do this and did not invest more time. I always ended up having node 6.x in the path and a quick google recommended using nvm on CircleCI

Reply:
Keep this in mind for future improvements